pkgs/games/steam/fhsenv.nix

```diff
@@ -23,9 +23,8 @@ let
     # Errors in output without those
     pciutils
     # run.sh wants ldconfig
-    glibc.bin
+    glibc_multi.bin
     # Games' dependencies
-    xorg.xrandr
```

Comment:
What's the argument for xrandr no longer being necessary? How do we know some games don't still depend on it? Or is it included with steam's runtime as well?

Reply:
I checked it against the flatpak and the Debian package and they both don't require it. If a specific game requires it, we should work with overwrites instead.

Reply:
If it's not in that flatpak, that is a pretty good indicator. Good enough for me.

Reply:
🤦🏼 derp, it is in the flatpak and they install it explcitily. Going to drop this again.